### ISSUE TITLE
extension on response causes unflushed io

### ISSUE THREAD

**Issue by benoist** [grip-framework/grip#9]
### Description

IO doesn't flush on close

### Steps to Reproduce

```crystal
class Index < Grip::Controller::Http
  def get(context)
    text(
      context,
      "-"*10000 + "!"
    )
  end
end

class Application < Grip::Application
  def initialize
    get "/", Index
  end
end

app = Application.new
app.run
```
**Expected behavior:** 

using curl http://localhost:3000/ the response should be closed and the ! should be returned

**Actual behavior:**

using curl http://localhost:3000/ the response is not closed and the ! is never printed

**Reproduces how often:** 

always

I narrowed it done to the extension on response
The original implementation shows
https://github.com/crystal-lang/crystal/blob/5999ae29b/src/http/server/response.cr#L222

But the grip redefined method does not flush the IO.
Looking at the methods they seem pretty similar, is this extension still required?

**Comment by grkek** [comment]
What version of the Grip framework are you using?

**Comment by benoist** [comment]
My apologies, I see it's fixed in 0.28.6
I was under the impression I was running master, turned I wasn't.
